ONSdigital/eq-terraform
# EQ-376 Author Infrastructure

**_What**_
1. Create a VPC for the author application
2. Create Subnets for the application and database
3. Create the Elastic beanstalk application for the author to deploy into
4. Create a Postgres database for the application
5. Create security group to whitelist IP to the ONS range.

I've also restructured the Survey Runner terraform files so that they all exist inside a survey-runner folder

**_How to test**_
1. Check out this branch
2. cd into author
3. Update the terraform.tfvars with the correct values for your environment
4. run `terraform apply`
5. Check you can hit the example application in elastic beanstalk.

**_Who can review**_
Anyone apart from @warrenbailey
